pkg/client/unversioned/restclient_test.go

```diff
@@ -286,3 +289,30 @@ func TestDoRequestCreated(t *testing.T) {
 	}
 	fakeHandler.ValidateRequest(t, "/"+testapi.Default.Version()+"/test", "GET", nil)
 }
+
+func TestCreateBackoffManager(t *testing.T) {
```

Comment:
Change from last PR iteration: This test is now just confirming backoff in an isolated context since we dont expose functionality for parsing parameters separately...  Sorta overlaps with request_test.go.... but I think its still good to keep around.